druid/examples/async_event.rs

```diff
@@ -0,0 +1,133 @@
+// Copyright 2020 The Druid Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! An example of sending commands from another thread.
+//! This is useful when you want to have some kind of
+//! generated content(like here), or some task that just
+//! takes a long time but don't want to block the main thread
+//! (waiting on an http request, some cpu intensive work etc.)
+
+use instant::Instant;
+use std::thread;
+use std::time::Duration;
+
+use druid::widget::prelude::*;
+use druid::{AppLauncher, Color, Selector, Target, WidgetExt, WindowDesc};
+
+// If you want to send commands to over event sinks you have to give it some kind
```

Comment:
`commands to over event sinks` not sure what you're saying here

Reply:
`commands over event sinks` is what I mean